apps/frontend/src/hooks/aave/utils.ts

```diff
@@ -0,0 +1,42 @@
+import { utils } from 'ethers';
+
+import { Decimal } from '@sovryn/utils';
+
+export const isBetweenZeroAndOne = (value: Decimal): boolean => {
+  const lowerBound = Decimal.from(0);
+  const upperBound = Decimal.from(1);
+
+  // Check if value is greater than or equal to 0 and less than or equal to 1
+  return value.gte(lowerBound) && value.lte(upperBound);
+};
+
+export const calculateUtilizationRate = (
```

Comment:
We're placing this ones in `utils/aave/AaveCalculations` and `totalDebt` and `AavailableLiquidity` should already be `Decimal` type from `import { Decimal } from '@sovryn/utils';` In case bignumbers are necessary we should use `BigNumber` from `import { BigNumber } from 'ethers';`